README.md

```diff
@@ -227,34 +227,79 @@ var playerConfig = {
+| `basePath` | It is `string` to represent the base path of the pdf file | Optional | Content will not load offline | It is required to load the pdf file in offline use case|
+| `baseDir` | It is `string` to represent the base path of the pdf file | Optional | Content will not load offline |  It is required to load the pdf file in offline use case |
+| `compatibilityLevel` | It is `number` to represent the compatibility level | Optional | Defalut compatibilityLevel 4 will set | Its an Optional field
+| `pkgVersion` | It is `number` to represent the version of the current packages | Optional | Defalut compatibilityLevel 1.0 will set | its an optional field
+ 
+  Sample config for mandatory fields
+```js
+var playerConfig = {
+	"metadata": {
+		identifier: 'do_31291455031832576019477',
+		name: 'NAME_OF_THE_CONTENT',
+		artifactUrl: 'https://ntpproductionall.blob.core.windows.net/ntp-content-production/content/assets/do_31291458881611366418883/b331332333_std_5_mathssciencesocial_tm_term-1_opt.pdf'
+    }	
+}
+```
+
 ## Telemetry property description
-|Property Name| Description| Default Value
-|--|----------------------|--|
-| `context` | It is an `object` it contains the `uid`,`did`,`sid`,`mode` etc., these will be logged inside the telemetry  | ```{}``` |
-| `mode` | It is  `string` to identify preview used by the user to play/edit/preview | ```play```|
-| `authToken` | It is  `string` and Auth key to make  api calls | ```''```|
-| `sid` | It is  `string` and User sessionid on portal or mobile | ```''```|
-| `did` | It is  `string` and Unique id to identify the device or browser| ```''```|
-| `uid` | It is  `string` and Current logged in user id| ```''```|
-| `channel` | It is `string` which defines channel identifier to know which channel is currently using.| `in.sunbird` |
-| `pdata` | It is an `object` which defines the producer information it should have identifier and version and canvas will log in the telemetry| ```{'id':'in.sunbird', 'ver':'1.0'}```|
-| `contextRollup` | It is an `object` which defines content roll up data | ```{}```|
-| `tags` | It is an `array` which defines the tag data | ```[]```|
-| `objectRollup` | It is an `object` which defines object rollup data | ```{}```|
-| `host` | It is  `string` which defines the from which domain content should be load|```window.location.origin```  |
-| `userData` | It is an `object` which defines user data | ```{}```|
-| `cdata` | It is an `array` which defines the correlation data | ```[]```|
+|Property Name| Description| Default Value | Mandatory/Optional| Comment|
+|--|----------------------|--|--| --|
+| `channel` | It is `string` which defines channel identifier to know which channel is currently using.| `in.sunbird` |Mandatory| Default can be set from player side.  Telemetry sdk will set by default ```“in.ekstep"```|
```

Comment:
Why user(Adaptor) need to know what player sets and what telemetry SDK, you mention what is the default value if not set